### ISSUE TITLE
Poor contrast in focused UI elements

### ISSUE THREAD

**Issue by Niektory** [widelands/widelands#4196]
Highlighting the focused UI elements right now decreases the contrast within them, causing decreased readability.

Some ideas how to improve it:
- Limit the area that is highlighted. In some cases too big of an area is highlighted when it's not necessary. For example highlight just a scrollbar instead of the contents that are scrolled.
- Indicate focus in some other way, for example draw a box around the focused widget. Might need some widget-specific adjustments.

**Screenshots:**
![shot0070](https://user-images.githubusercontent.com/4470900/91773006-399fc980-ebe6-11ea-8384-dddd088e6612.png)![shot0071](https://user-images.githubusercontent.com/4470900/91773010-3d335080-ebe6-11ea-850d-9d19c82f3887.png)
![shot0073](https://user-images.githubusercontent.com/4470900/91773015-402e4100-ebe6-11ea-8025-8bfffcc54abe.png)![shot0072](https://user-images.githubusercontent.com/4470900/91773012-3efd1400-ebe6-11ea-8607-1f6eec86330a.png)

**Version:**
1.0~git24831[3c8ef56@master]

**Comment by gunchleoc** [comment]
I'd be OK with either of these solutions:

1. Draw a border
2. Make background image darker so we'll have more range for contrast

For scrollbars and buttons, I guess just making them lighter should be sufficient.
